Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM ubuntu:20.04
+FROM ruby:3.2.2
```

Comment:
@lwrubel @edsu This OK? It's only used for dev/test, right?

Reply:
Yep!